nibabel/nicom/dicomwrappers.py

```diff
@@ -467,6 +467,25 @@ def __init__(self, dcm_data):
             self.shared = dcm_data.get('SharedFunctionalGroupsSequence')[0]
         except TypeError:
             raise WrapperError('SharedFunctionalGroupsSequence is empty.')
+        # Try to determine slice order and minimal image position patient
+        self._frame_slc_ord = self._ipp = None
+        try:
+            frame_ipps = [f.PlanePositionSequence[0].ImagePositionPatient for f in self.frames]
```

Comment:
The only way I imagine this would come up is if someone did some manual editing ended up butchering the DICOM file. At least the current code should error out in this case (either in the constructor or when someone tries to use any functionality that needs IPP).